.github/workflows/static_checks.yml

```
@@ -26,7 +26,7 @@ jobs:
 
           - name: clazy
```

Nit: Add cppcheck here
```suggestion
          - name: "clazy, cppcheck"
```

---

I can't change this without adding another config key, since I use the name for the preset, and I need a build to run before cppcheck because it also checkes moc files that are part of the json file.

---

Hm, okay, we could add another key for the preset, but let's do this if it becomes more of an issue.